src/components/Modal/index.jsx

```diff
@@ -198,7 +198,7 @@ export default class Modal extends React.Component {
     }
 
     return (
-      <Portal closeOnEsc isOpened onClose={this.cancel}>
```

Comment:
Ok, what happen with `this.cancel`? We solve this on other places with an instance variable https://github.com/factorialco/factorial-frontend/blob/develop/src/components/ImportModal/index.jsx#L22